reviewrot/gitlabstack.py

```diff
@@ -46,7 +46,7 @@ def request_reviews(self, user_name, repo_name=None, age=None,
         try:
             gl.auth()
         except SSLError as e:
-            log.exception('Error during authentification: %s', e.message)
```

Comment:
This used to cause an error in python3 ('AttributeError: 'SSLError' object has no attribute 'message''). Changing it to __str__() fixes this.

Reply:
Does `str(e)` also work here? (instead of `e.__str__()`)

Reply:
Yes, I've just changed it to str(e)